port/win/port_win.h

```diff
@@ -241,6 +241,17 @@ extern void InitOnce(OnceType* once, void (*initializer)());
 #define CACHE_LINE_SIZE 64U
 #endif
 
+// To reduce false cache sharing by enforcing alignment
+#if defined(__clang__) || defined(__GNUC__)
+# define CACHE_ALIGNED(size) alignas(size)
```

Comment:
Tried, but alignas() of win still throws the same warning which causes compile error. :(